public/static/partials/user_host_details.html

```diff
@@ -86,7 +86,10 @@ <h2 style="display:inline-block; padding-right:15px">
             <strong>Expiration:</strong>
             <md-datepicker  ng-disabled="curHostData.no_expiration" ng-model="curHostData.current_expiration" md-min-date="curHostData.original_expiration" md-placeholder="Date"></md-datepicker>
             <md-time-picker read-only="curHostData.no_expiration"   ng-model="curHostData.current_expiration" class="inline" message="message"></md-time-picker>
-            <md-checkbox id="no-expiration" class="inline" ng-show="curHostData.host_type == 'ec2-ondemand'" ng-model="curHostData.no_expiration" ng-click="setCurrentExpirationOnClick()" style="margin-left:25px">Never</md-checkbox>
+            <md-checkbox id="no-expiration" class="inline" ng-show="curHostData.host_type == 'ec2-ondemand'" ng-model="curHostData.no_expiration" ng-disabled="!unexpirableEnabled()" ng-click="setCurrentExpirationOnClick()" style="margin-left:25px">Never</md-checkbox>
+            <em ng-class="{'text-muted': unexpirableEnabled(), 'text-error': !unexpirableEnabled()}" style="margin-left: 5px;" ng-cloak>
```

Comment:
I would ng-show this using the same conditions as above, otherwise you'll have intent hosts with this text but no checkbox

Reply:
done